setup.py

```diff
@@ -84,7 +84,6 @@ def run_tests(self):
         'Natural Language :: English',
         'License :: OSI Approved :: Apache Software License',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2.6',
```

Comment:
Good that this one was removed, but there should exist a trove classifier for `'Programming Language :: Python :: 2'` as requests still support Python 2 in some capacity.